integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/insertQuery/InsertIntoNonCarbonTableTestCase.scala

```diff
@@ -0,0 +1,109 @@
+      "ActiveCheckTime string, ActiveAreaId string, ActiveCountry string, ActiveProvince string, " +
+      "Activecity string, ActiveDistrict string, ActiveStreet string, ActiveOperatorId string, " +
+      "Active_releaseId string, Active_EMUIVersion string, Active_operaSysVersion string, " +
+      "Active_BacVerNumber string, Active_BacFlashVer string, Active_webUIVersion string, " +
+      "Active_webUITypeCarrVer string,Active_webTypeDataVerNumber string, Active_operatorsVersion" +
+      " string, Active_phonePADPartitionedVersions string, Latest_YEAR int, Latest_MONTH int, " +
+      "Latest_DAY Decimal(30,10), Latest_HOUR string, Latest_areaId string, Latest_country " +
+      "string, Latest_province string, Latest_city string, Latest_district string, Latest_street " +
+      "string, Latest_releaseId string, Latest_EMUIVersion string, Latest_operaSysVersion string," +
+      " Latest_BacVerNumber string, Latest_BacFlashVer string, Latest_webUIVersion string, " +
+      "Latest_webUITypeCarrVer string, Latest_webTypeDataVerNumber string, " +
+      "Latest_operatorsVersion string, Latest_phonePADPartitionedVersions string, " +
+      "Latest_operatorId string, gamePointDescription string,gamePointId double,contractNumber " +
+      "BigInt) STORED BY 'org.apache.carbondata.format'")
+    sql(
+      s"LOAD DATA INPATH '$resourcesPath/100_olap.csv' INTO table TCarbonSource options " +
+      "('DELIMITER'=',', 'QUOTECHAR'='\', 'FILEHEADER'='imei,deviceInformationId,MAC,deviceColor," +
+      "device_backColor,modelId,marketName,AMSize,ROMSize,CUPAudit,CPIClocked,series," +
+      "productionDate,bomCode,internalModels,deliveryTime,channelsId,channelsName,deliveryAreaId," +
+      "deliveryCountry,deliveryProvince,deliveryCity,deliveryDistrict,deliveryStreet," +
+      "oxSingleNumber,ActiveCheckTime,ActiveAreaId,ActiveCountry,ActiveProvince,Activecity," +
+      "ActiveDistrict,ActiveStreet,ActiveOperatorId,Active_releaseId,Active_EMUIVersion," +
+      "Active_operaSysVersion,Active_BacVerNumber,Active_BacFlashVer,Active_webUIVersion," +
+      "Active_webUITypeCarrVer,Active_webTypeDataVerNumber,Active_operatorsVersion," +
+      "Active_phonePADPartitionedVersions,Latest_YEAR,Latest_MONTH,Latest_DAY,Latest_HOUR," +
+      "Latest_areaId,Latest_country,Latest_province,Latest_city,Latest_district,Latest_street," +
+      "Latest_releaseId,Latest_EMUIVersion,Latest_operaSysVersion,Latest_BacVerNumber," +
+      "Latest_BacFlashVer,Latest_webUIVersion,Latest_webUITypeCarrVer," +
+      "Latest_webTypeDataVerNumber,Latest_operatorsVersion,Latest_phonePADPartitionedVersions," +
+      "Latest_operatorId,gamePointDescription,gamePointId,contractNumber', " +
+      "'bad_records_logger_enable'='false','bad_records_action'='FORCE')")
+  }
+
+  test("insert into hive") {
+    sql("drop table if exists thive2")
+    sql(
+      "create table thive2 row format delimited fields terminated by '\017' stored as textfile as" +
+      " " +
```

Comment:
remove this, and format the statement into proper line wrap. 
``` 
    CREATE TABLE xxx
    ROW FORMAT DELIMITED ...
    STORED AS ...
    AS
        SELECT ...
```

Reply:
@jackylk is there any option to add these rules in scala style?

Reply:
I do not think there are such rules, since it is SQL specific

Reply:
done